test/Templates.Test/Helpers/WebDriverFactory.cs

```diff
@@ -24,13 +26,20 @@ public static IWebDriver CreateWebDriver()
             var result = IsAppVeyor ? CreateFirefoxDriver() : CreateEdgeDriver();
             result.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(1);
             return result;
+
+            IWebDriver CreateFirefoxDriver()
+            {
+                return WebDriverFactory.CreateFirefoxDriver(new FirefoxOptions(){
```

Comment:
I've validated that this works with latest firefox

Reply:
Great. Minor nitpick comment - braces look non-C#-ish :)